Hat details

mays2pearl opened the pull request.

elenadiniz committed:
Ambiente de Desenvolvimento

- Criação do Ambiente de Desenvolvimento, contendo algumas das framaworks utilizadas no projeto já em funcionamento.
- Necessária atualização de algumas controllers para deixa-las como no antigo projeto.

elenadiniz committed:
Update de Pods e atualização de Layout

- Incluido o Google.plist e a framework Firestore.
- Refatorado alguns recursos de layout para garantir que todos os itens não quebrem dentro da Navigation Bar.
- Teste com todas as frameworks feitos e todos os dados estão retornando normalmente.

elenadiniz committed:
Correção de Layout

- Corrigido Layout da Tela de Questões do Quiz e inserido novamente o scrollView, assim como nas telas de detalhe dos personagens e dos animais fantásticos.

elenadiniz committed:
Update

elenadiniz committed:
Fetch User Data from FirebaseAuth

- Inicio da implementação do método para trazer os dados do usuário, salvos no FirebaseAuth.

elenadiniz committed:
Correção de Layout quebrando na TabBar e continuação da implementação do getUser

elenadiniz committed:
Features de implementação do User Firebase

elenadiniz committed:
Adicionado novos pods ao arquivo PodFile e iniciado recuperação dos dados pelo Firebase

elenadiniz committed:
Update nos métodos que populam as telas de detalhes dos personagens e animais fantásticos.

elenadiniz committed:
Remoção de warning

elenadiniz committed:
Atualização de layout conforme diretrizes do Firebase e implementação do método para alterar a imagem de perfil

elenadiniz committed:
Adicionando Firebase RealtimeDatabase e FirebaseStorage

- Criado método para salvar a imagem escolhida pelo usuário no Database do Firebase.

elenadiniz committed:
Update método salvar dados no Storage e no Firebase Database

- Finalizada a implementação do método para salvar a imagem no Storage do Firebase.
- Iniciada a implementação para passar o valor do Storage para o Database e salvar também os demais dados do usuário (nome completo, data de nascimento e país).

elenadiniz committed:
Pod's update

elenadiniz committed:
Inserido método nas telas de alteração de senha

elenadiniz committed:
Update de layout de telas de senha e finalização dos métodos que enviam o email de redefinição para o usuário

mays2pearl committed:
implementando sorteio do chapeu

mays2pearl committed:
melhorias no chapeu seletor

elenadiniz reviewed (changes requested):
Oriento que a Label criada para mostrar a casa, fique escondida antes do sorteio e que quando ele (sorteio) for finalizado, o TextField fique escondido e a Label apareça com a casa sorteada. 

Como exemplo para a orientação acima, sugiro olhar a tela de RecoveryPassword, pois ela possui esse mesmo comportamento e o código já está pronto.

mays2pearl committed:
melhorias e ajustes tela Hat

Inclusão de validação  de preenchimento de campo, ativação do botão para a ação especifica do sorteio.

mays2pearl committed:
Incluido a casa no Usuario

Incluido o sorteio da casa na controller de configurações do usuario, nome e label de acordo com a casa sorteada.

elenadiniz commented:
Já margeado.
